src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -262,6 +262,205 @@ def test_timestamp_included(self):
 
         self.assertEqual(reference_node.text, "Report created on: Jan. 1, 2024, 1 a.m.")
 
+    def test_textfield_component_with_multiple_is_rendered_as_html(self):
+        submission = SubmissionFactory.from_components(
+            [
+                {
+                    "type": "textfield",
+                    "key": "textfield-single",
+                    "label": "Textfield single",
+                    "multiple": False,
+                },
+                {
+                    "type": "textfield",
+                    "key": "textfield-multiple",
+                    "label": "Textfield multiple",
+                    "multiple": True,
+                },
+            ],
+            submitted_data={
+                "textfield-single": "foo",
+                "textfield-multiple": ["foo", "bar"],
+            },
+            with_report=True,
+        )
+        html = submission.report.generate_submission_report_pdf()
+        print(html)
```

Comment:
Can you remove the `print` statements to avoid the test output clutter?

Reply:
Ah check, yeah that shouldn't be there... I'll remove it 👍